website/docs/reference/login-history.md

```diff
@@ -0,0 +1,42 @@
+---
+title: Login history
+---
+
+:::info
+
+Login history is an enterprise feature available from Unleash 4.22 onwards.
+
+:::
+
+Unleash's login history lets you track login events in your Unleash instance, and whether the attempts were successful in logging in or not. 
+
+![Login history table](/img/login-history-table.png)
+
+For each login event, it lists:
+
+ - **Created**: When it happened
+ - **Username**: The username that was used
+ - **Authentication**: The authentication type that was used
+ - **IP address**: The IP address that made the attempt
+ - **Success**: Whether the attempt was successful or not
+ - **Failure reason**: If the attempt was not successful, the reason why
+
+You can see the failure reason by hovering over the "False" badge in the "Success" column.
+
+![Login history table failure reason](/img/login-history-table-fail.png)
+
+Use the login history to:
+
+- Audit login events in your Unleash instance
+- Identify failed login attempts and investigate the cause
+- Debug misconfigured authentication providers
+
+You're able to download the current login history at any time. If you'd like, you can also delete single login events, or clear the login history by deleting all the events.
+
+In order to access and download the existing login history in your Unleash instance, you can follow the [how to download login history](../how-to/how-to-download-login-history.mdx) guide.
```

Comment:
Nice! But maybe we can rephrase it a bit? How about something like this?

```suggestion
The login history is mutable: You can [remove individual login attempts or clear the entire history](../how-to/how-to-download-login-history.mdx#deleting-login-history-events) by deleting all of them.

Finally, the login history can be downloaded ([how do I download my Unleash login history](../how-to/how-to-download-login-history.mdx)) for external backups, audits, and the like.
```

I think that presents the same information in a clearer manner.

Reply:
I like it, but now I'm unsure of the importance of having the deletion section: https://github.com/Unleash/unleash/pull/3261#discussion_r1129445483